plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -1083,10 +1083,9 @@ def _addSettingElement(self, builder, instance):
             # Skip material properties (eg diameter) or metadata (eg GUID)
             return
 
-        if instance.value is True:
-            data = "yes"
-        elif instance.value is False:
-            data = "no"
+        truth_map = { True: "yes", False: "no" }
```

Comment:
I'm a big fan of this variable name :rofl:

Reply:
I wasn't sure what else to call it 😅